packages/block-library/src/social-link/edit.native.js

```diff
@@ -119,6 +119,7 @@ const SocialLinkEdit = ( {
 				toValue: 1,
 				duration: ANIMATION_DURATION,
 				easing: Easing.circle,
+				useNativeDriver: false,
```

Comment:
When using `Animated`, it is required to specify the value of `useNativeDriver`. This change suppresses the LogBox warning:

```
Animated: useNativeDriver was not specified. This is a required option and must be explicitly set to true or false.
```